app/presenters/hyrax/work_show_presenter.rb

```diff
@@ -1,5 +1,8 @@
+require 'action_view'
+
 module Hyrax
   class WorkShowPresenter
+    include ActionView::Helpers::SanitizeHelper
```

Comment:
Instead of including the entire `SanitizeHelper`, could we use `Loofah` directly?